src/cast_core/include/cast_core/actors/RunCommand.hpp

```diff
@@ -76,7 +72,8 @@ class RunCommand : public Actor {
     genny::DefaultRandom _rng;
     mongocxx::pool::entry _client;
 
-    PhaseLoop<PhaseState> _loop;
+    struct PhaseConfig;
```

Comment:
I don't think I'm ever going to agree with calling these things configs. If it has a run method, it's not a config. Still, not gonna die on a molehill.

Reply:
ya I see what you're saying - but `State` also didn't feel right because it's not the state of the current phase or anything, it's a pre-configured per-phase mini-actor-lite thing...maybe `PhaseAction` or something? idk. open to other names but we should be consistent in what our Actors call it to make it less confusing for new Actor-authors.